manifests/Spacefarer.yaml

```diff
@@ -0,0 +1,18 @@
+name: Spacefarer
+authors: samrocketman and Airhead
+homepage: https://github.com/samrocketman/Spacefarer
+license: GPL-3.0
+version: b3d3ffa06d931b111435169929f535970152b4a6
+shortDescription: One Jump Drive; No ship capturing; No plundering installed outfits.
+description: As a captain, you're responsible for the lives of your crew.  Loss of
+  life is inevitable in the merciless reaches of space.  You stick to exploration,
+  ferrying, and fleet combat.  If anyone dies, it will be everyone when the ship explodes.  This
+  plugin adds replay value by forcing the player to discover alternate and competing
+  forms of income in order to efficiently obtain a fleet.
```

Comment:
Are the double spaces in this paragraph intentional?

Reply:
This is what autoupdate.py generated.  The original description was one line.

I added a few minimal bits and let autoupdate.py run on it to fill out the version bits.

Reply:
I was referring to the punctuation - I'd disagree if this was collective writing, but I just wanted to make sure you didn't make an error. Your description, your rules.